### DISCUSSION TITLE
Accumulating PRs

### DISCUSSION THREAD

**Discussion by luisdavim** [neoclide/coc.nvim#4862] [General]
I see a lot of PRs from @fannheyward accumulating, is there no one to review/approve them?

**Comment by fannheyward** [comment]
Thank you for your question, at least 1 approving review is required by reviewers with write access for the PR to merge, and chemzqm is busy recently, that's why there're some PRs didn't get merged yet.

I've merged these fixes and features PRs to `dev`, you can use `Plug 'neoclide/coc.nvim', {'branch': 'dev', 'do': 'npm ci'}` to use.

**Reply by luisdavim** [reply]
Thanks for the update.
